zerver/tests/test_events.py

```diff
@@ -2469,34 +2469,31 @@ def do_set_realm_property_test(self, name: str) -> None:
         vals = test_values.get(name)
         property_type = Realm.property_types[name]
         if property_type is bool:
-            validator: Validator[object] = check_bool
             vals = bool_tests
-        elif property_type is str:
-            validator = check_string
-        elif property_type == (str, type(None)):
-            validator = check_string
-        elif property_type is int:
-            validator = check_int
-        elif property_type == (int, type(None)):
-            validator = check_int
-        else:
-            raise AssertionError(f"Unexpected property type {property_type}")
-        schema_checker = check_events_dict([
-            ('type', equals('realm')),
-            ('op', equals('update')),
-            ('property', equals(name)),
-            ('value', validator),
-        ])
 
         if vals is None:
             raise AssertionError(f'No test created for {name}')
+
         do_set_realm_property(self.user_profile.realm, name, vals[0])
         for val in vals[1:]:
             state_change_expected = True
             events = self.verify_action(
                 lambda: do_set_realm_property(self.user_profile.realm, name, val),
                 state_change_expected=state_change_expected)
-            schema_checker('events[0]', events[0])
+
+            realm_update_schema('events[0]', events[0])
```

Comment:
I'm not thrilled with this naming scheme; `check_realm_update_schema` or `check_realm_update_event` would work for me and be in line with how name other similar checker functions.  But it's totally unclear on seeing it what `realm_update_schema` is supposed to do -- does it return the schema?  Or compute it?  Or what.

Reply:
I'm not a big fan of `check_realm_update`, as I kinda wanna leave the `check_foo` namespace for non-Zulip-specific checks.

Options:
* realm_update_verify
* realm_update_check
* check_realm_update
* verify_realm_update

My preference is still `realm_update_schema`.  It's a schema that happens to be invoked as a function.